src/window/window_wrapper/keyboard_manager.rs

```diff
@@ -58,10 +58,13 @@ impl KeyboardManager {
             } => {
                 // Record the modifer states so that we can properly add them to the keybinding
                 // text
-                self.shift = modifiers.shift_key();
-                self.ctrl = modifiers.control_key();
-                self.alt = modifiers.alt_key();
-                self.logo = modifiers.super_key();
+                if !self.ignore_input_this_frame {
```

Comment:
Better to use `guard clause`,

```
if condition {
  return;
}
```

Reply:
If you prefer, ok.

Though, I find it a bit dangerous to jump right out of the function. People adding code after `match event {}` later might scratch their heads ;)

Reply:
Oh, i haven't seen it was inside match event.

I was thinking that was just plain function. In that case we should leave it as it is. (Or maybe split into function, like `self.handle_modifiers_change`.